x-pack/test_serverless/functional/test_suites/common/management/index_management/index_detail.ts

```diff
@@ -33,9 +33,12 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       await pageObjects.indexManagement.clickCreateIndexSaveButton();
       await pageObjects.indexManagement.expectIndexToExist(testIndexName);
     });
-    it('index with no documents', async () => {
-      await pageObjects.indexManagement.indexDetailsPage.openIndexDetailsPage(0);
-      await pageObjects.indexManagement.indexDetailsPage.expectIndexDetailsPageIsLoaded();
+    describe('can view index details', function () {
+      this.tags(['skipSvlSearch']);
```

Comment:
can you also add a test to the serverless search index details to test this functionality? ie that clicking the index in the index list table opens the new index details page.

Reply:
Yes. But we should also enable search_indices in search config.ts. As an example added in this commit 2497572ee26377073046614d4728062727ad2986

Reply:
Moved index management details page logic test to [search_index_detail](https://github.com/elastic/kibana/blob/f9dbbe5d17e37c758da5403c4b94e497e3b9ec3a/x-pack/test_serverless/functional/test_suites/search/search_index_detail.ts) 
commit: https://github.com/elastic/kibana/pull/194857/commits/7eb76ddb00914509ca871377654a06244178caac